### DISCUSSION TITLE
[Feature Request] Make the date board work with the Tasks plugin

### DISCUSSION THREAD

**Discussion by ushham** [roovo/obsidian-card-board#72] [Ideas]
It would be great if the user could choose whether this plugin used the Tasks plugin due date format, or the date of the note title

**Comment by roovo** [comment]
I've got a couple of other itches I want to scratch first but I do plan to have a look into this!   My thinking is to define something generic which can be configured to work with the Task format.

**Comment by roovo** [comment]
Hiya @ushham, the just released [version 0.6.0](https://github.com/roovo/obsidian-card-board/releases/tag/0.6.0) has added support for Due and Completion dates in Tasks format.  (hopefully) better late than never :)

**Reply by fabiobruna** [reply]
This works perfect. Thank you!

**Reply by roovo** [reply]
any probs, just let me know!

**Comment by ushham** [comment]
Great update, thank you!
